packages/scandipwa/src/component/ProductActions/ProductActions.container.js

```diff
@@ -322,15 +322,16 @@ export class ProductActionsContainer extends PureComponent {
         } = customPrice;
 
         const selectedOptions = this.getSelectedOptions();
-        const prices = options.reduce((acc, { data }) => {
-            data.forEach(({ option_type_id, price }) => {
+        const prices = options
+            .flatMap(({ data }) => data)
```

Comment:
I don't think you really need a `flatMap` here. Semantically, this is very complex to understand. 
Please use something more straightforward, e.g. `map` or `reduce` here.